python/lsst/pipe/tasks/postprocess.py

```diff
@@ -278,18 +280,7 @@ class WriteSourceTableConnections(pipeBase.PipelineTaskConnections,
 
 class WriteSourceTableConfig(pipeBase.PipelineTaskConfig,
```

Comment:
Is this just left here for compatibility reasons? Otherwise I don't see why it wouldn't be better to just remove it.

Reply:
Is the question why we need a `writeSourceTableConfig` that doesn't have any config fields?  Remember that the Task's config class holds the connections, which are configurable:
```  
writePreSourceTable:
    class: lsst.pipe.tasks.postprocess.WriteSourceTableTask
    config:
      connections.outputCatalog: preSource
```